hack/update-versions.sh

```diff
@@ -62,12 +62,12 @@ if [ $NEW_SHORT != $OLD_SHORT ]; then
     cp "$newfile" "$oldfile"
     class_version=${OLD_SHORT//.}
     class_name="Kn"
-    sed -i -r "s/class\s+${class_name}\s+<\s+Formula/class ${class_name}AT${class_version} < Formula/" "$oldfile"
+    sed -i '' -r "s/class\s+${class_name}\s+<\s+Formula/class ${class_name}AT${class_version} < Formula/" "$oldfile"
```

Comment:
:see_no_evil: 

I love `sed` :sweat_smile:

Reply:
It's the greatest love story of all times, different `sed` version on different OS. :)

Reply:
Perl is the same on every OS .... just saying :-P  

And yes you can easily replace `sed` with `perl`:

```
perl -ipe "s/class\s+$ENV{class_name}\s+<\s+Formula/class $ENV{class_name}AT$ENV{class_version} < Formula/" "$oldfile"
```
Just nagging here a bit, happy with the PR as is ...

Reply:
@dsimansk maybe add a comment above why to add `''` with an external link maybe ? Might help other people, too and make it understandable for the casual reader ...

Reply:
Sure, I'll add a comment.